CRISPResso2/CRISPRessoWGSCORE.py

```diff
@@ -302,10 +315,26 @@ def print_stacktrace_if_debug():
 
         args = parser.parse_args()
 
+        CRISPRessoShared.set_console_log_level(logger, args.verbosity, args.debug)
+
         crispresso_options = CRISPRessoShared.get_crispresso_options()
         options_to_ignore = {'fastq_r1', 'fastq_r2', 'amplicon_seq', 'amplicon_name', 'output_folder', 'name', 'zip_output'}
         crispresso_options_for_wgs = list(crispresso_options-options_to_ignore)
 
+        OUTPUT_DIRECTORY='CRISPRessoWGS_on_%s' % normalize_name(args)
```

Comment:
may be better to explicitly pass in bam_file and name (from args) in case we need to change these arg names in the future